src/vs/workbench/contrib/extensions/browser/extensionsViews.ts

```diff
@@ -423,6 +423,12 @@ export class ExtensionsListView extends ViewletPanel {
 			options.sortBy = SortBy.InstallCount;
 		}
 
+		// {{SQL CARBON EDIT}}
+		if (/@visualizerExtensions/i.test(query.value)) {
```

Comment:
can this be done more generically using categories?  @alanrenmsft any ideas?  this doesn't seem scalable as a solution for extension recommendations.